sysapi/sysapi/Tss2_Sys_AC_GetCapability.c

```diff
@@ -0,0 +1,104 @@
+/***********************************************************************;
+ * Copyright (c) 2017, Intel Corporation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ ***********************************************************************/
+
+#include "sapi/tpm20.h"
+#include "sysapi_util.h"
+
+TSS2_RC Tss2_Sys_AC_GetCapability_Prepare(
+    TSS2_SYS_CONTEXT *sysContext,
+    TPMI_RH_AC ac,
+    TPM_AT capability,
+    UINT32 count)
+{
+    if (!sysContext)
+        return TSS2_SYS_RC_BAD_REFERENCE;
+
+    CommonPreparePrologue(sysContext, TPM_CC_AC_GetCapability);
+
+    Marshal_UINT32(SYS_CONTEXT->tpmInBuffPtr, SYS_CONTEXT->maxCommandSize,
+                  &SYS_CONTEXT->nextData, ac, &SYS_CONTEXT->rval);
```

Comment:
The alignment of the second line isn't consistent with the way you align similar things below. Your `&` is under the `(` instead of aligned with the first parameter.

Reply:
Fixed